src/utils/functionRuntimeUtils.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as opn from 'opn';
+import * as semver from 'semver';
+import * as vscode from 'vscode';
+import { extensionPrefix } from '../../src/ProjectSettings';
+import { DialogResponses } from '../DialogResponses';
+import { localize } from '../localize';
+import { cpUtils } from './cpUtils';
+
+export namespace functionRuntimeUtils {
+
+    const runtimePackage: string = 'azure-functions-core-tools';
+
+    export async function validateFunctionRuntime(outputChannel: vscode.OutputChannel): Promise<void> {
+        try {
+            const localVersion: string | null = await getLocalFunctionRuntimeVersion();
+            if (localVersion === null) {
+                return;
+            }
+            const newestVersion: string | null = await getNewestFunctionRuntimeVersion(semver.major(localVersion));
+            if (newestVersion === null) {
+                return;
+            }
+            if (semver.gt(newestVersion, localVersion)) {
+                await promptDocumentationAction(
+                    localize(
+                        'azFunc.outdatedFunctionRuntime',
+                        '[Azure Functions] Your installed version of the Azure Functions Core Tools ("{0}") does not match the latest ("{1}"). Please update for the best experience.',
```

Comment:
Hmm this message is still pretty long and will get cutoff from the user (although they can view the tooltip)

![screen shot 2018-02-07 at 8 25 50 am](https://user-images.githubusercontent.com/11282622/35927915-be2581c4-0be0-11e8-80bc-cd262b919054.png)

Here are my suggestions for shortening it:
1. Remove '[Azure Functions]'
1. Remove 'installed'
1. Remove the quotes inside the parentheses

So the final version would look like this:

'Your version of the Azure Functions Core Tools ({0}) does not match the latest ({1}). Please update for the best experience.'